app/helpers/spelling_suggestions_helper.rb

```diff
@@ -51,9 +51,8 @@ def render_suggestion(original_url, search, suggested_query, suggested_url, spel
     showing_results_for = t :showing_results_for, corrected_query: suggested_query_link
     search_instead_for = t :search_instead_for, original_query: original_query_link
 
-    render partial: 'searches/spelling_suggestion.html',
+    render partial: 'searches/spelling_suggestion',
```

Comment:
We should also fix this in the `render_spelling` method:

```ruby
  def render_spelling(suggested_query, suggested_url, module_tag)
...
    render partial: 'searches/spelling_correction.html',
           locals: { showing_results_for: showing_results_for.html_safe}
  end
```
We unfortunately don't have unit tests for that method, so you won't see the deprecation warning unless you run the cucumber tests.